src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,33 @@ Master Pre-Releases
 This document describes ``dev`` releases which occur weekly from master, and which do
 not undergo the vetting associated with ``stable`` releases.
 
+1.10.0rc0 (09/10/2018)
+----------------------
+
```

Comment:
This should already be in here somewhere (before the 1.11.0 dev series started).

Reply:
But also, the `rc0` release should be in here.

Reply:
Oops, this should be an 11 with a new date. Fixed.